src/components/select/select.js

```diff
@@ -316,20 +315,23 @@ function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $rootElement,
       element.attr(ariaAttrs);
 
       scope.$on('$destroy', function() {
-        if (isOpen) {
-          $mdSelect.hide().finally(function() {
```

Comment:
Do we know why the `hide()` used to be called? Did we lose an animation by skipping this, or was it irrelevant?